molecule/driver/digitalocean.py

```diff
@@ -0,0 +1,138 @@
+#  Copyright (c) 2015-2018 Cisco Systems, Inc.
```

Comment:
Should be `... Red Hat ...`, please see the other recently edited files.

Reply:
Included in lastest commit 👍